src/tests/plugins/VuePlugin.test.ts

```diff
@@ -0,0 +1,61 @@
+import { VuePlugin, RawPlugin } from "../../index";
+import { createEnv } from "../stubs/TestEnvironment";
+import { should } from "fuse-test-runner";
+
+const vueFileSource = `<template>
+    <div>
+        <p>{{ msg }}</p>
+        <input type="text" v-model="msg" />
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'app',
+        data () {
+            return {
+                msg: 'Welcome to Your Vue.js App'
+            }
+        }
+    }
+</script>
+`;
+
+export class VuePluginTest {
+    "Should return compiled vue code with render functions"() {
+        return createEnv({
+            project: {
+                files: {
+					"app.vue": vueFileSource
+                },
+                plugins: [
+                    [ VuePlugin(), RawPlugin({})]
+                ],
+                instructions: ">app.vue",
+            },
+        }).then((result) => {
+			const out = result.project.FuseBox.import("./app.vue").trim();
+			should(out).findString('_p.render');
+			should(out).findString('_p.staticRenderFns');
+			should(out).notFindString('_p.template');
+
+			should(out).equal(`var _p = {};
+var _v = function(exports){"use strict";
```

Comment:
I couldn't agree more. We were already discussing a better way. Let me see if I can improve it today